.github/workflows/build_wheels_linux.yml

```
@@ -55,6 +55,16 @@ on:
         description: "The key created when saving a cache and the key used to search for a cache."
         default: ""
         type: string
+      architecture:
+        description: Architecure to build for x64 for default Linux, or ARM64 for Linux aarch64 builds
```

s/Architecure/Architecture/
Also, I propose to use `x86_64` rather than `x64`, to match results of `uname -m` call on respective hosts
```suggestion
        description: Architecture to build for x86_64 for default Linux, or aarch64 for Linux aarch64 builds
```

---

Sounds good @malfet let me do this!